invision-forum-crawler.py

```diff
@@ -80,7 +80,8 @@
 DATASET_NAME = "dataset_name" # for example: forum_website_pl_corpus
 DATASET_DESCRIPTION = f"Collection of forum discussions from {DATASET_URL}" # also change this according to the forum name
 LICENSE = "(c) www.forumaddress.pl" # forum address
-EXPECTED_URL_PARTS = ['/topic','/temat', '/thread'] # we're targeting the full topics to optimize the performance
+
+EXPECTED_URL_PARTS = ['/topic','/temat', '/thread', '/forum'] # we're targeting the full topics to optimize the performance
```

Comment:
Is the '/forum' part needed? The script does only go through the topics, not whole forums/subforums.

Reply:
Found forums where early topics were under '/forum'